### DISCUSSION TITLE
How to create keyboard shortcut to debug/run current test

### DISCUSSION THREAD

**Discussion by majkinetor** [pester/Pester#2342] [Q&A] [answered]
Current test is the one my cursor is into. The effect would be the same as if I clicked Run/Debug above the test, which is highly unproductive to do all the time for me.

**Comment by fflaten** [comment, marked as answer]
I'd start with a feature request in https://github.com/PowerShell/vscode-powershell

This requires a vscode extension to get the startline for `It`-keyword. Easy to create as a new extension, but easier to extend the existing one if they want the feature. 🙂

**Reply by majkinetor** [reply]
I did that here: https://github.com/PowerShell/vscode-powershell/issues/4529

Thanks.
